cookbooks/aws-parallelcluster-install/recipes/install.rb

```diff
@@ -22,14 +22,29 @@
 
 node_attributes 'Generate export in json at /etc/chef/node_attributes.json'
 
+# == PLATFORM - BASE
 include_recipe 'aws-parallelcluster-install::base'
+
+# == PLATFORM - FEATURES
+include_recipe "aws-parallelcluster-install::nvidia" # NVIDIA and CUDA
+include_recipe "aws-parallelcluster-install::intel_mpi" unless virtualized?
+include_recipe "aws-parallelcluster-install::cloudwatch_agent"
```

Comment:
this should be in the "observability" group.

Reply:
Will fix later.